products/apache-activemq.md

```diff
@@ -17,6 +17,10 @@ identifiers:
   - repology: activemq
   - cpe: cpe:/a:apache:activemq
   - cpe: cpe:2.3:a:apache:activemq
+  - purl: pkg:maven/org.apache.activemq/activemq-broker
+  - purl: pkg:maven/org.apache.activemq/activemq-pool
+  - purl: pkg:maven/org.apache.activemq/activemq-spring
+  - purl: pkg:maven/org.apache.activemq/activemq-jdbc-store
```

Comment:
```suggestion
  - purl: pkg:maven/org.apache.activemq/activemq-broker
```

> I added them, because we have applications which use a embedded active mq. Same goes for tomcat, which also has an embedded one.

Got it, in this case I would suggest only keeping `activemq-broker` ?